config/sync/core.entity_form_display.paragraph.paragraph_internal.default.yml

```diff
@@ -0,0 +1,24 @@
+uuid: f4925b26-f29b-452f-a035-4f2d7698a6e3
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.paragraph_internal.field_internal_content
+    - paragraphs.paragraphs_type.paragraph_internal
```

Comment:
O nome não está me agradando muito. External and Internal

Externo o que? interno o que?

O que acha de apenas content_reference e custom_reference? Qualquer coisa valida com o Reva

Reply:
ok!